Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json

```diff
@@ -9,9 +9,9 @@
                 "updated": 1628037249031
             },
             {
-                "language": "RU",
-                "text": "Иерархия Plugins определяет, какие плагины должны быть включены в текущую рабочее пространство Workspace. В контексте этой иерархии плагины - это файлы JSON, содержащие описание иерархии, такой как Data Mine, Trading Mine, Trading Engine или Trading System.\n",
-                "updated": 1639552216965
+                "language": "EL",
+                "text": "Η ιεραρχία των πρόσθετων καθορίζει ποιά πρόσθετα θα συμπεριληφθούν στον τρέχοντα χώρο εργασίας. Στο πλαίσιο αυτής της ιεραρχίας, τα πρόσθετα είναι αρχεία JSON που διαθέτουν την περιγραφή μιας ιεραρχίας όπως ένα εξορυκτήριο δεδομένων, ένα εξορυκτήριο συναλλαγών, μια μηχανή συναλλαγών ή ένα σύστημα συναλλαγών.",
+                "updated": 1641828795260
```

Comment:
You accidentally overwrite the Russian translation. I assume that this happened unintentionally. Do you know how to fix the problem? If not, please ask, then we can help you. This applies to all paragraphs in this file.

Reply:
Hi mate and thank you for your constructive comments. This problem occurs many times and i have no clue why it is happening nor how to fix it. Any suggestion will be much appreciated

Reply:
Hey @theobserver this problem occurs when multiple people work on one File. For example when someone is translating a file into spanish and someone else into russian. The best way to avoid this problem is to very often app.update and commit. app.update keeps everything up to date and the more "smaller" commits and pull request the faster we can merge. :+1: